sdk/formrecognizer/azure-ai-formrecognizer/src/samples/README.md

```diff
@@ -29,8 +29,8 @@ All of these samples need the endpoint to your Form Recognizer resource ([instru
 |----------------|-------------|
 |[Authentication][authentication_sample]|Authenticate the client|
 |[RecognizeContent][recognize_content] and [RecognizeContentFromUrlAsync][recognize_content_from_url_async]|Recognize text and table structures of a document|
-|[RecognizeIdDocuments][recognize_id_documents] and [RecognizeIdDocumentsAsync][recognize_id_documents_async]|Recognize data from an ID document like a passport or a US drivers license using a prebuilt model|
```

Comment:
ohh interesting, the other prebuilt endpoints are not here (business cards, invoices). I know it is not part of your PR, so you don't have to do it here. Either way, could you make sure it gets done/issue filed?

Reply:
Agreed. I created an issue to address it. https://github.com/Azure/azure-sdk-for-java/issues/21139